src/technologies/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: article
+title: "Technologies"
+description: "Find out what our technology stacks are"
+tags: [home, dev]
+order: 
+  home: 40
+  dev: 100
+related:
+  tag: tech
+---
+## Languages and frameworks
+
+We use these languages:
+
+* __Java and Spring__
```

Comment:
Worth mentioning Project Lombok along with the Java tech stack?

Reply:
Its mentioned in the Java tech page